changelog/0.8.2/pull-1552

```diff
@@ -0,0 +1,12 @@
+Feature: Use Google Application Default credentials
```

Comment:
This file needs to be moved to `changelog/0.8.3` now, we've released 0.8.2 yesterday :)